moveit_planners/chomp/chomp_motion_planner/src/chomp_optimizer.cpp

```diff
@@ -35,18 +35,22 @@
 /* Author: Mrinal Kalakrishnan */
 
 #include <chomp_motion_planner/chomp_optimizer.h>
-#include <ros/ros.h>
-#include <visualization_msgs/MarkerArray.h>
 #include <chomp_motion_planner/chomp_utils.h>
+#include <moveit/planning_scene/planning_scene.h>
 #include <moveit/robot_model/robot_model.h>
 #include <moveit/robot_state/conversions.h>
-#include <moveit/planning_scene/planning_scene.h>
-#include <eigen3/Eigen/LU>
+
 #include <eigen3/Eigen/Core>
+#include <eigen3/Eigen/LU>
 #include <random>
+#include <visualization_msgs/msg/marker_array.hpp>
+
+#include "rclcpp/rclcpp.hpp"
 
 namespace chomp
 {
+rclcpp::Logger LOGGER2 = rclcpp::get_logger("chomp_optimizer");
```

Comment:
```suggestion
rclcpp::Logger LOGGER = rclcpp::get_logger("chomp_optimizer");
```
Why did you number the loggers?